src/models/Domain.php

```diff
@@ -110,6 +110,7 @@ public function rules()
                 'disable-hold',
                 'enable-w-p-freeze',
                 'disable-w-p-freeze',
+                'notify-transfer-in',
```

Comment:
[['id'], 'required', 'on' => [
                'enable-freeze',
                'disable-freeze',
                'sync',
                'regen-password',
                'set-note',
                'set-autorenewal',
                'set-whois-protect',
                'set-lock',
                'push-with-pincode',
                'enable-hold',
                'disable-hold',
                'enable-w-p-freeze',
                'disable-w-p-freeze',
                'notify-transfer-in',                                                                                                                                                                                           
            ]]